dashboard/app/bug_impact_score.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2025 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import "regexp"
+
+// impactOrder represent an ordering of bug impact severity. The earlier
+// entries are considered more severe.
+// The regexps are matched in order, and the first match determines the
+// impact category. A higher index in this slice means lower impact.
+// The last regexp ".*" is a catch-all for unknown bugs.
+var impactOrder = []string{
+	// Memory safety.
+	`^KASAN:.*Write`,
+	`^KASAN:.*Read`,
+	`^WARNING: refcount bug`,
+	`^UBSAN: array-index`,
+	`^BUG: corrupted list`,
+	`^BUG: unable to handle kernel paging request`,
+	// Memory leaks.
+	`^memory leak`,
+	// Uninit memory use.
+	`^KMSAN:`,
+	// Concurrency.
+	`^KCSAN:`,
+	// Locking.
+	`^BUG: sleeping function`,
```

Comment:
I am thinking if this should be above BUG/NULL derefs...
cc @melver @ramosian-glider @FlorentRevest thoughts on exact ordering?

Reply:
Note, problem with recent CVEs is that they hardly distinguish severity anymore. Older CVEs likely were a stronger indicator of severity, but there also were fewer of those.

Reply:
Generally speaking we're sharing the intuition here. But I don't think it'll work for us long-term.
In 3 years it'll be quite strange to prioritize bugs by 5 years old CVE statistics.

As an alternative @FlorentRevest proposed to summarize the intuition we used here on the documentation page.
That documentation page may be used to explain the priorities rationale and at the same time may be used as a basement for the priorities adjustment requests.
I expect it to work better for us long-term.

Reply:
You can find the documentation page in this PR.